src/server/job_func.c

```diff
@@ -422,21 +422,16 @@ job_alloc(void)
 
 
 #ifndef PBS_MOM
-	/* mark as JOB_INITIAL, set accrue times to zero */
-	pj->ji_wattr[(int)JOB_ATR_accrue_type].at_val.at_long = JOB_INITIAL;
-	pj->ji_wattr[(int)JOB_ATR_eligible_time].at_val.at_long = 0;
```

Comment:
I don't think you can get rid of this.  update_eligible_time() adds to the existing eligible time.  pj was malloc()'d so this is garbage.